chrome/content/zotero/components/virtualized-table.jsx

```diff
@@ -1062,10 +1063,12 @@ class VirtualizedTable extends React.Component {
 			Zotero.debug("Attempting to update virtualized-table font size with a prop-specified rowHeight."
 				+ "You should change the prop on the React component instead");
```

Comment:
`updateFontSize()` is intended to be called only when `fontSize` pref is changed on trees that have font size scaling enabled, otherwise if a row height change is expected it should be done manually by changing the prop. So this is a warning for future ourselves to not expect anything to happen. But the if statement is no longer correct above and the logged message needs to be altered. 

This method also probably needs a comment for further clarification.